app/controllers/tasks_controller.rb

```diff
@@ -13,6 +13,12 @@ def index
         render json: Task.all
     end
 
+    def destroy
+        task = Task.find(params[:id])
+        task.destroy
+        render json: "The task was successfully deleted."
```

Comment:
🥸  I spy some duplication in this file pertaining to `render json` is there any way we can DRY up this controller a bit and add a JSON Presenter helper class?

Reply:
I added a JsonPresenterHelper.